IMC_DataScience/IMC_OOPTutorial/blog.md

```diff
@@ -47,25 +49,34 @@ To start, let’s make a public class called “Person” (since that’s the ty
 
 ![img](https://lh6.googleusercontent.com/zehLJy0dg7jY0SIAesHgX1_GRqjn_et0uq-QoJPPNZWSta4vbNkB1AlJnqcahxzgZHD-pG96n0ZiGzIKNM0W2T6cbDEwZ256ye3eybvMx-EX5dhS1-eXaI5Yuc48F4uLhsRLb-zf)
 
-To clarify, we’re making this object in its own file. To actually create a “person” object, we’ll just leave the “Person.java” file in the same directory as whatever program is actually using the object, then create an object in that file. How do we actually create the object, though? Well, that uses something we call a **constructor**. A constructor is a bit of code in the object class (the Person.java file) that lets you know how exactly you’re going to create a Person object in your code. Here’s a sample one I wrote:
```

Comment:
For the image above, you could show the closing brackets in this, since in C++ they use a semicolon at the end bracket to signify the end of a class while Java doesn't (just a small thing tho)